eth/api.go

```diff
@@ -32,12 +32,14 @@ import (
 	"github.com/ethereum/go-ethereum/core/state"
 	"github.com/ethereum/go-ethereum/core/types"
 	"github.com/ethereum/go-ethereum/core/vm"
+	vmlogger "github.com/ethereum/go-ethereum/core/vm"
```

Comment:
Any particular reason you imported the `vm` a second time with a different alias?